ast/ddl.go

```diff
@@ -71,12 +73,25 @@ const (
 	DatabaseOptionCharset
 	DatabaseOptionCollate
 	DatabaseOptionEncryption
+	DatabaseOptionPlacementPrimaryRegion
+	DatabaseOptionPlacementRegions
+	DatabaseOptionPlacementFollowerCount
+	DatabaseOptionPlacementVoterCount
+	DatabaseOptionPlacementLearnerCount
+	DatabaseOptionPlacementSchedule
+	DatabaseOptionPlacementConstraints
+	DatabaseOptionPlacementLeaderConstraints
+	DatabaseOptionPlacementLearnerConstraints
+	DatabaseOptionPlacementFollowerConstraints
+	DatabaseOptionPlacementVoterConstraints
+	DatabaseOptionPlacementPolicy
```

Comment:
how about defining the db options to have the same values as the table options like

```go
    DatabaseOptionPlacementPrimaryRegion = DatabaseOptionType(TableOptionPlacementPrimaryRegion)
    DatabaseOptionPlacementRegions = DatabaseOptionType(TableOptionPlacementRegions)
    // etc
```

...